### ISSUE TITLE
Update .fec file submission status messages

### ISSUE THREAD

**Issue by mjtravers** [fecgov/fecfile-web-api#571]
The backend is sending submission status messages that do not reflect the state of the submission.

There are 2 situations in the period of time during the submission of a .fec file that is not being properly represented by the messages being sent back to the user

1. A state of failure in the backend service of fecfile online after the user submits the .fec file but before the .fec file is sent to the FEC submission API.
2. A state where the backend service of fecfile online is taking a long time to process but before the .fec file is sent to the FEC submission API.

The fecfile online backend should be updated to send to the front-end an clear and appropriate message for each of these 2 states.

**DEV NOTES**
The code to update is in the reports app view in the get_status_mapping() function.
Any error messages thrown by a backend exception should be recorded in the fecfile_error column of the upload_submissions database table.

**Comment by AureliaKhorsand** [comment]
@mjtravers will this have anything to do with the submission errors the users will see on the front end when we implement that?

**Comment by mjtravers** [comment]
@AureliaKhorsand Yes, but these are situations where things go wrong on our side and not error codes we could possibly receive from the EFO service when we submit a .fec file. We need to incorporate errors from our system into the message process that we will be setting up to convert error codes/messages we get from EFO into user-friendly messages we show to the user.

**Comment by GreggMoreland** [comment]
@AureliaKhorsand should this ticket be added to the submission error ticket group? This is something I think we need to include. Currently, the user will get either a Submission Pending or a Submission Failure and no EFO generated messages.
